include/AudioEngine.h

```diff
@@ -398,25 +398,27 @@ class LMMS_EXPORT AudioEngine : public QObject
 
 	bool m_isProcessing;
 
-	// audio device stuff
+	// audio device
 	void doSetAudioDevice( AudioDevice *_dev );
 	AudioDevice * m_audioDev;
 	AudioDevice * m_oldAudioDev;
 	QString m_audioDevName;
 	bool m_audioDevStartFailed;
 
-	// MIDI device stuff
+	// Metronome
+	bool m_metronomeActive;
+	std::pair<tick_t, tick_t> m_prevMetronomeStates; // .first: number of ticks from last play sound, .second: number of ticks from last call
```

Comment:
Could we place the comment on the line above so it doesn't extend so far out like this.

Reply:
- [ ] move comments